### ISSUE TITLE
Curly brackets in comment

### ISSUE THREAD

**Issue by nesetril** [lanl/dfnWorks#31]
If there is this comment:

```
regions:
{-5, 5, -10, 10, -20, 10} // {minX, maxX, minY, maxY, minZ, maxZ}

```
then {minX, maxX, minY, maxY, minZ, maxZ} shall be read instead of {-5, 5, -10, 10, -20, 10}

The error is: ValueError: could not convert string to float: 'minX'

**Comment by hymanjd** [comment]
Thank you for pointing this out. What file is the error in?

**Comment by nesetril** [comment]
/pydfnworks/dfnGen/generation/input_checking/parsing.py

line 318                   	float(value[0]),

**Comment by hymanjd** [comment]
Thank you. I mean the input file path.

**Comment by nesetril** [comment]
It is not in an example file available online. I made this in my dfnGen file.

**Comment by hymanjd** [comment]
okay. Thank you for letting me know.


______________________________________
Jeffrey Hyman
Computational Earth Science (EES-16)
Los Alamos National Laboratory
1 - (505) - 665 - 2074
***@***.***
________________________________
From: KamCZa ***@***.***>
Sent: Saturday, June 18, 2022 9:40:59 AM
To: lanl/dfnWorks
Cc: Hyman, Jeffrey De'Haven; Comment
Subject: [EXTERNAL] Re: [lanl/dfnWorks] Curly brackets in comment (Issue #31)


It is not in an example file available online. I made this in my dfnGen file.

**Comment by hymanjd** [comment]
this was fixed by moving to v2.7
